drivers/modem/wncm14a2a.c

```diff
@@ -500,7 +504,7 @@ static int pkt_setup_ip_data(struct net_pkt *pkt,
 
 #if defined(CONFIG_NET_UDP)
 	if (sock->ip_proto == IPPROTO_UDP) {
-		if (net_udp_create(pkt, src_port, dst_port)) {
+		if (net_udp_create(pkt, dst_port, src_port)) {
```

Comment:
Did we decide the order here was ok?  Note that the TCP header is also created the same way as here.  So if this is wrong, there's a very good chance TCP is wrong as well?

Reply:
I did verify this was needed in order to fix issues with UDP/DTLS.  If I left this the way it was, DTLS does not work;
Agree, I'll fix the TCP one as well.